MAUI/DataGrid/Column-types.md

```diff
@@ -0,0 +1,916 @@
+
+{% highlight xaml %}
+<syncfusion:SfDataGrid.Columns>
+    <syncfusion:DataGridTextColumn="OrderID">
+        <syncfusion:DataGridTextColumn.HeaderTemplate>
+            <DataTemplate>
+                <Label x:Name="OrderID" Text="OrderID" TextColor="Black" 
+                       BackgroundColor="Yellow"/>
+            </DataTemplate>
+        </syncfusion:DataGridTextColumn.HeaderTemplate>
+    </syncfusion:DataGridTextColumn>
+</syncfusion:SfDataGrid.Columns> 
+{% endhighlight %}
+
+### Setting manual column width
+
+SfDataGrid allows you to customize the width of each DataGridColumn in the `SfDataGrid.Columns` collection. To customize column width, use the `DataGridColumn.Width` property. By default, this property will not be assigned any value. The DataGridColumn renders in view based on the value of the `DefaultColumnWidth` property.
+
+N> Set the `Visible` property to `True` instead of setting column width as `0` to hide a column.
+
+Customize the width for auto generated columns in both XAML and code as follows:
+
+{% tabs %}
+{% highlight xaml %}
+<sfgrid:SfDataGrid x:Name="dataGrid"
+                   AutoGenerateColumnsMode="None"
+    <sfgrid:SfDataGrid.Columns x:TypeArguments="sfgrid:Columns">
+      <sfgrid:DataGridTextColumn MappingName="OrderID"
+                             Width="100"/>
+      </sfgrid:SfDataGrid.Columns > 
+  </sfgrid:SfDataGrid>
+  
+{% endhighlight %}
+{% highlight c# %}
+// AutoGenerated Column
+
+dataGrid.AutoGeneratingColumn += DataGrid_AutoGeneratingColumn;
+
+void dataGrid_AutoGeneratingColumn(object sender, AutoGeneratingColumnArgs e){
+if (e.Column.MappingName == "OrderID") {
+    e.Column.Width = 100;
+    }
+}
+
+// Manually generated column
+
+dataGrid.Columns.Add(new DataGridTextColumn() { MappingName = "OrderID" ,Width = 100 });
+
+{% endhighlight %}
+{% endtabs %}
+
+### Hiding a column 
+
+To hide a particular column, use the `DataGridColumn.Visible` property. The default value of the `Visible` property is `True`. 
+
+N> Set the `Visible` property to `False` instead of setting column width as `0` to hide a column.
+
+To hide column using the `Visible` property, follow the code example:
+
+{% tabs %}
+{% highlight xaml %}    
+    <syncfusion:DataGridTextColumn MappingName="OrderID" Visible = "False"/>
+{% endhighlight %}
+{% highlight c# %}
+// AutoGenerate Column
+
+dataGrid.AutoGeneratingColumn += DataGrid_AutoGeneratingColumn;
+
+void dataGrid_AutoGeneratingColumn(object sender, AutoGeneratingColumnArgs e){
+if (e.Column.MappingName == "OrderID") {
+    e.Column.Visible = false;
+    }
+}
+
+// Manually generated column
+
+dataGrid.Columns.Add(new DataGridTextColumn() { MappingName = "OrderID", Visible = false});
+{% endhighlight %}
+{% endtabs %}
+
+### Padding
+
+SfDataGrid allows the users to set padding for the Header and cells in display mode by using the property `DataGridColumn.HeaderPadding` and `DataGridColumn.CellPadding`. 
```

Comment:
The SfDataGrid allows users to set padding for the Header and cells in display mode by using the property `DataGridColumn.HeaderPadding` and `DataGridColumn.CellPadding`.